Frontend/src/App.jsx

```diff
@@ -26,7 +26,9 @@ function App() {
     { id: 4, name: "Dr. Brown", specialty: "Neurology", patients: 100, appointments: 350, experience: 18, qualifications: "MD, PhD", image: "https://randomuser.me/api/portraits/women/4.jpg", bio: "Dr. Brown is a leading neurologist with expertise in treating a wide range of neurological disorders and conducting groundbreaking research." },
     { id: 5, name: "Dr. Taylor", specialty: "Dermatology", patients: 180, appointments: 550, experience: 8, qualifications: "MD, FAAD", image: "https://randomuser.me/api/portraits/men/5.jpg", bio: "Dr. Taylor is a skilled dermatologist specializing in both medical and cosmetic dermatology, helping patients achieve healthy and beautiful skin." },
     { id: 6, name: "Dr. Alex", specialty: "Gynecologist", patients: 350, appointments: 650, experience: 10, qualifications: "MD, PhD", image: "https://randomuser.me/api/portraits/men/94.jpg", bio: "Dr. Alex is an experienced gynecologist providing comprehensive women's health care, specializing in reproductive health, pregnancy care, and preventive medicine for women of all ages."},
-    { id: 7, name: "Dr. Wilson", specialty: "General Medicine", patients: 250, appointments: 700, experience: 20, qualifications: "MD, FACP", image: "https://randomuser.me/api/portraits/women/6.jpg", bio: "Dr. Wilson is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
+    { id: 7, name: "Dr. Wilson", specialty: "General practitioner", patients: 250, appointments: 620, experience: 12, qualifications: "MD, PHD", image: "https://randomuser.me/api/portraits/women/6.jpg", bio: "Dr. Wilson is an experienced general practitioner providing comprehensive primary care and preventive medicine for patients of all ages." },
```

Comment:
Please update the descriptions for the doctors you added, as all of them currently have the same description.

Reply:
okk i will do it..